libunwind/src/assembly.h

```diff
@@ -15,6 +15,7 @@
 #ifndef UNWIND_ASSEMBLY_H
 #define UNWIND_ASSEMBLY_H
 
+#ifndef __wasm__
```

Comment:
Again, this is not OK.

First, this file just defines macros, so even under WASM it should be safe to include.
If you're hitting the `#error` case in the `#else`, then add an `#elif __wasm__`.

While his style of code may not cause issues for you, because you don't use it. It makes things more complicated for every change that actually uses these files.